Playwright config path issue in Git Bash on Windows

### Description

Here is an error message.
`Error: Cannot find module '/C:/Users/Sandesh/Local%20Sites/gutenberg/app/public/wp-content/plugins/gutenberg/test/e2e/config/global-setup.ts'`

Full screenshot

![a6818792-507d-4111-9205-b540cc3dce21](https://user-images.githubusercontent.com/15683894/159926154-43a7061c-9a30-4e18-a44b-f988080e077d.jpg)


--------------------
Quick fix  - I have changed 2 lines in  `gutenberg\test\e2e\playwright.config.ts` to fix it temporary

`testDir: new URL( './specs', 'file:' + __filename ).pathname,` => `testDir: './specs',`

`globalSetup: new URL( './config/global-setup.ts', 'file:' + __filename ).pathname,` => `globalSetup: './config/global-setup.ts',`

### Step-by-step reproduction instructions

1. Start test env
2. Run `npm run test-e2e:playwright`
3. You will face this issue in Git Bash on windows

### Screenshots, screen recording, code snippet

_No response_

### Environment info

- Laptop with Windows 10
- Git Bash
- Node 14.16

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

--- a/test/e2e/playwright.config.ts
+++ b/test/e2e/playwright.config.ts
@@ -2,6 +2,7 @@
  * External dependencies
  */
 import path from 'path';
+import { fileURLToPath } from 'url';
 import { devices } from '@playwright/test';
 import type { PlaywrightTestConfig } from '@playwright/test';
 
@@ -19,10 +20,11 @@ const config: PlaywrightTestConfig = {
 	timeout: parseInt( process.env.TIMEOUT || '', 10 ) || 100_000, // Defaults to 100 seconds.
 	// Don't report slow test "files", as we will be running our tests in serial.
 	reportSlowTests: null,
-	testDir: new URL( './specs', 'file:' + __filename ).pathname,
+	testDir: fileURLToPath( new URL( './specs', 'file:' + __filename ).href ),
 	outputDir: path.join( process.cwd(), 'artifacts/test-results' ),
-	globalSetup: new URL( './config/global-setup.ts', 'file:' + __filename )
-		.pathname,
+	globalSetup: fileURLToPath(
+		new URL( './config/global-setup.ts', 'file:' + __filename ).href
+	),
 	use: {
 		baseURL: process.env.WP_BASE_URL || 'http://localhost:8889',
 		headless: true,

